src/w3c/templates/headers.js

```diff
@@ -3,11 +3,25 @@ import showLogo from "./show-logo";
 import showPeople from "./show-people";
 import showLink from "./show-link";
 
+function decorateSpecTitle(specTitleElem) {
+  specTitleElem.classList.add("title", "p-name");
+  specTitleElem.id = "title";
+}
+
 export default conf => {
   const html = hyperHTML;
+  const specTitleElem = document.querySelector("h1#title, body>h1") || document.createElement("h1");
```

Comment:
We should actually just require `"h1#title"`.  That is, to use this, Editors should explicitly write: `<h1 id="title">Foo</h1> `. Other `h1` should be ignored.